common/lib/xmodule/xmodule/capa_module.py

```diff
@@ -1724,7 +1724,7 @@ def submit_problem(self, data, override_time=False):
                 return {'success': msg, 'html': ''}
 
         # Wait time between resets: check if is too soon for submission.
-        if self.last_submission_time is not None and self.submission_wait_seconds != 0:
+        if self.last_submission_time is not None and self.submission_wait_seconds not in [0, None]:
```

Comment:
Entirely optional nit: Both `0` and `None` evaluate to False, so you could also do:

```python
if self.last_submission_time is not None and self.submission_wait_seconds:
```

But I think explicit is fine too.

Reply:
I thought about that, but kept explicit for readability, namely being explicit on what we are excluding